examples/custom-controller/README.md

```diff
@@ -0,0 +1,5 @@
+# Custom Controller for Agones Game Servers
+
+This is an example of how to write a customer controller integrated with Agones resources.
+
+This example shows how to create, deploy and run a Custom Kubernetes Controller for Agones that logs changes to GameServers and modifies their labels.
```

Comment:
As per the other ones, this should link to the documentation page we're writing now.

Reply:
I've made all the suggested changes.